src/components/Toaster/Provider/ToasterProvider.tsx

```diff
@@ -60,20 +60,28 @@ export const ToasterProvider = React.forwardRef<ToasterPublicMethods, Props>(
             [],
         );
 
+        const toastsRef = React.useRef<InternalToastProps[]>();
+        toastsRef.current = toasts;
```

Comment:
React documentation recommends not writing or reading `ref.current` during rendering https://beta.reactjs.org/reference/react/useRef#caveats